main.py

```diff
@@ -26,15 +26,20 @@
 
 import firestore.admins
 from common import User
-from firestore.incidents import (deleteIncident, getIncidents, getStats,
-                                 insertIncident)
+from firestore.incidents import deleteIncident, getIncidents, getStats, insertIncident
+from firestore.tokens import registerNewToken
 import incident_publisher
+
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
+
 # [END gae_python3_datastore_store_and_fetch_user_times]
 # [END gae_python38_datastore_store_and_fetch_user_times]
 app = Flask(__name__)
 # cors = CORS(app, resources={r"/*": {"origins": "*"}})
 cors = CORS(app)
 firebase_request_adapter = requests.Request()
+limiter = Limiter(get_remote_address, app=app, default_limits=["200/day", "50/hour"])
```

Comment:
We can commit for now.
But we need to do more research to make sure this is the right way.

Limit on the same remote address might not be appropriate. For example, many people behind the same firewall might share the same public IP. Is there a way to get mobile device id ?